docs/src/numerical_implementation/pressure_decomposition.md

```diff
@@ -1,39 +1,62 @@
     \end{equation}
 ```
-The anomalous hydrostatic component of the kinematic pressure is defined by 
+
+The hydrostatic pressure component in \eqref{eq:pressure} is defined so that the vertical
+component of its gradient balances gravity:
+
+```math
+    \begin{align}
+    \partial_z p_{\text{total hydrostatic}} & = - g \left ( 1 + \frac{\rho_*}{\rho_0} + \frac{\rho'}{\rho_0} \right ) \, ,
+    \end{align}
+```
+
+Above, we use the notation introduced in the [Boussinesq approximation](@ref boussinesq_approximation)
+section.
+
+We can further split the hydrostatic pressure component into
 ```math
     \begin{align}
-    \label{eq:hydrostaticpressure}
-    \partial_z p_{\rm{hyd}} \equiv -b \, ,
+    p_{\text{total hydrostatic}}(\boldsymbol{x}, t) = p_{*}(z) + p_{\rm{hyd}}(\boldsymbol{x}, t) \, ,
     \end{align}
 ```
-such that the sum of the kinematic pressure and buoyancy perturbation becomes
+
+i.e., a component that only varies in ``z`` (``p_*``) and a "hydrostatic anomaly" (``p_{\rm{hyd}}``) defined
+so that
+
 ```math
     \begin{align}
-    -\boldsymbol{\nabla} p + b \boldsymbol{\hat z} = 
-        - \boldsymbol{\nabla} p_{\rm{non}}
-        - \boldsymbol{\nabla}_h p_{\rm{hyd}} \, ,
+    \partial_z p_{*} & = - g \left ( 1 + \frac{\rho_*}{\rho_0} \right ) \, ,\\
+    \partial_z p_{\rm{hyd}} & = \underbrace{- g \frac{\rho'}{\rho_0}}_{= b} \, .
     \end{align}
 ```
-where ``\boldsymbol{\nabla}_h \equiv \partial_x \boldsymbol{\hat x} + \partial_y \boldsymbol{\hat y}`` 
-is the horizontal gradient. The hydrostatic pressure anomaly is so named because the "total" 
-hydrostatic pressure contains additional components:
+
+Doing so, the gradient of the kinematic pressure becomes:
+
 ```math
-\begin{align}
-\partial_z p_{\text{total hydrostatic}} & = - g \left ( 1 + \frac{\rho_*}{\rho_0} + \frac{\rho'}{\rho_0} \right ) \, , \\
-                                           & = \partial_z p_{\rm{hyd}} - g \left ( 1 + \frac{\rho_*}{\rho_0} \right ) \, .
-\end{align}
+    \begin{align}
+    \boldsymbol{\nabla} p & = \boldsymbol{\nabla} p_{\rm{non}} + \boldsymbol{\nabla}_h p_{\rm{hyd}} + ( \partial_z p_{*} + \partial_z p_{\rm{hyd}} ) \boldsymbol{\hat z}\, ,
+    \end{align}
 ```
-Under this pressure decomposition the pressure gradient that appears in the momentum equations becomes
+
+where ``\boldsymbol{\nabla}_h \equiv \boldsymbol{\hat x} \partial_x +  \boldsymbol{\hat y} \partial_y``
+is the horizontal gradient.
+
+Under this pressure decomposition, the kinematic pressure gradient that appears in the momentum equations
+(after we've employed the the [Boussinesq approximation](@ref boussinesq_approximation)) combines with
+the gravity force to give:
+
 ```math
-   \boldsymbol{\nabla} p \mapsto \boldsymbol{\nabla} p_{\rm{non}} + \boldsymbol{\nabla}_h p_{\rm{hyd}}\, .
+    \begin{align}
+    \boldsymbol{\nabla} p + g \frac{\rho}{\rho_0} \hat {\boldsymbol{z}} = \boldsymbol{\nabla} p_{\rm{non}} + \boldsymbol{\nabla}_h p_{\rm{hyd}} \, .
```

Comment:
Should we be using buoyancy at some point instead of density?

Reply:
Well it’s introduced above. But here the point is that gravity force plus pressure gradient disappears in place of hydrostatic + non hydrostatic  pressure gradients.